pkgs/data/themes/juno/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib, stdenv, fetchurl, gtk-engine-murrine }:
+
+stdenv.mkDerivation rec {
+  pname = "juno";
+  version = "0.0.1";
+
+  srcs = [
+    (fetchurl {
+      url =
+        "https://github.com/gvolpe/Juno/releases/download/${version}/Juno.tar.xz";
```

Comment:
```suggestion
      url = "https://github.com/gvolpe/Juno/releases/download/${version}/Juno.tar.xz";
```

Reply:
Is this a joke? :stuck_out_tongue_closed_eyes: I updated this after @humancalico 's suggestion to use `nixfmt`. Is `nixfmt` required or not? I honestly don't like it.

Reply:
@gvolpe lol I have no idea. I just use it and then follow  @SuperSandro2000's suggestions :rofl:

Reply:
@SuperSandro2000 can you tell us how you get that format? I applied `nixfmt` and that's not the result I get.

Reply:
Maybe this https://github.com/nix-community/nixpkgs-fmt

Reply:
I usually use nixpkgs-fmt but it tends to add to many line breaks in lists. nixfmt tends to enforce 80 line length in sub optimal places.

Reply:
Is there any guideline published about the use of formatters? Last time I contributed this was not a thing and I find it very confusing.

Reply:
It is not required to use a formatter but it makes your live easier to format the file well and except the line breaks I accept nixfmt and nixpkgs-fmt.

Reply:
Then I don't understand why you would suggest changes to use `nixpkgs-fmt` after it's been formatted using `nixfmt`, seems pedantic :man_shrugging:

Reply:
because nixfmt enforces short line lengths which add ugly spaces like this one.

Reply:
So you say "it is not required to use a formatter" and it seems there's no official guideline about it so we just need to roll with whatever seems "ugly" to you? What you consider to be "ugly" might not be so to others. That is the literal definition of being pedantic.